src/chords.js

```diff
@@ -703,7 +704,382 @@ export const chords = [
         2: 3
       }
     }
-  }
+  },
+// End of guitar chords
+
+// Ukulele chords
```

Comment:
This file is already mega huge. What do you think about splitting it into 2 files for each instrument? The file structure could be

```
src/chords/guitar.js
src/chords/ukulele.js
```

Reply:
Sure, will do, but then we would need to do the filtering twice in `App.js`, storing the chords in two different variables. Or should we concatenate all of the still in the same variable `chords`?

Reply:
Maybe for now just leave it and we can refactor it later 👍